### PR TITLE
fix log message and optimize log format check logic

### PR DESCRIPTION
**Pull request by pandaamanda**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by fedebongio** [comment]
Hi! We triaged this one, could you please fill the description so we have a better idea of the changes we are looking at? thanks!

**Comment by pandaamanda** [comment]
> Hi! We triaged this one, could you please fill the description so we have a better idea of the changes we are looking at? thanks!

Updated, thanks.

File: pkg/kubeapiserver/options/authentication.go [hunk 1]
```diff
@@ -217,9 +217,8 @@ func (o *BuiltInAuthenticationOptions) Validate() []error {
 	}
 
 	if o.WebHook != nil {
-		retryBackoff := o.WebHook.RetryBackoff
-		if retryBackoff != nil && retryBackoff.Steps <= 0 {
-			allErrors = append(allErrors, fmt.Errorf("number of webhook retry attempts must be greater than 1, but is: %d", retryBackoff.Steps))
+		if o.WebHook.RetryBackoff != nil && o.WebHook.RetryBackoff.Steps <= 0 {
+			allErrors = append(allErrors, fmt.Errorf("number of webhook retry attempts must be greater than 0, but is: %d", o.WebHook.RetryBackoff.Steps))
```
Comment (caesarxuchao) [inline pkg/kubeapiserver/options/authentication.go:221]: I think the original code is better. The chain of referring is hard to read.

**Review by caesarxuchao — COMMENTED** [review]
Otherwise lgtm

File: pkg/kubeapiserver/options/authentication.go (hunk 1 above)
Comment (pandaamanda) [reply pkg/kubeapiserver/options/authentication.go]: ok, fixed.

**Comment by pandaamanda** [comment]
@liggitt  kindly ping for milestone v1.21

**Comment by liggitt** [comment]
this isn't release blocking, it can merge into 1.22 once the milestone opens

**Pull request merged**